test/battle/item_effect/restore_pp.c

```diff
@@ -66,3 +66,28 @@ SINGLE_BATTLE_TEST("Max Elixir restores the PP of all of a battler's moves fully
 }
 
 TO_DO_BATTLE_TEST("Ether won't work if the selected move has all its PP")
+
+SINGLE_BATTLE_TEST("Elixir can be used if at least one move is missing PP in any slot") // As it works right now, this test can't test what it's suppossed to, since the relevant check doesn't run in the testing system.
```

Comment:
If the test is broken (because of the underlying system), I would personally just mark it as to do

Reply:
Commented out the test body and changes to `TO_DO_BATTLE_TEST`